functions.py

```diff
@@ -0,0 +1,111 @@
+import time
+import turtle
+import random
+
+score = 4
+score_a = 0
+score_b = 0
```

Comment:
This variables are not constants. They are getting changed by the outside code. It is a bad design.

Imagine, you decided to allow user playing this game multiple times within the same process. In this case, whenever the game is loaded for the second time, `score_a` and `score_b` will have the values saved from the previous game.